server/views/index.html

```diff
@@ -102,6 +102,22 @@
 
             });
         </script>
+
+        <!-- Soletta Dev-App header -->
+        <div class="headerDevapp">
+            <div style="width:245px; display: flex;">
+                <div class="boxTitle" style="width:40px; margin-left: -15px;"></div>
+                <div class="headerSolettaLogo">
+                    <img src="imgs/soletta_logo.png" style="margin-top: 6px; margin-bottom: 5px;"/>
```

Comment:
mmm.... since you're distributing these image with logo and trademark I believe you may need to check licenses to go with it, don't you think.

@anselmolsm could you give us any advice about it?